engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/CreateProcessInstanceTest.java

```diff
@@ -198,4 +198,24 @@ public void shouldTakeSequenceFlowFromStartEvent() {
         .hasBpmnProcessId("process")
         .hasProcessInstanceKey(processInstanceKey);
   }
+
+  @Test
+  public void shouldCompleteAbstractTask() {
+    // given
+    ENGINE.deployment().withXmlClasspathResource("/processes/abstract_task_process.bpmn").deploy();
```

Comment:
🔧 It would be cool if we could replace this with out builder-style process creation. This makes it a lot easier to see what the deployed process looks like, instead of having to open this file first.

```java
    ENGINE
        .deployment()
        .withXmlResource(
            Bpmn.createExecutableProcess("process")
                .startEvent()
                .abstractTask() // Optionally add an id
                .sequenceFlowId("flow")
                .endEvent()
                .done())
        .deploy();
```

This is not possible out of the box currently. Some changes would have to be made in the model to support the abstract task here.
I have not marked this as a blocker as it is not the most important part of the PR and we could easily do this in a different issue. Please let me know if you'd be interested in adding this as part of this PR, or if you want me to create a new issue.

Reply:
👍 
Hi @remcowesterhoud. 
Just now @saig0  and I came to the conclusion to add a `.task()` method, I will submit the corresponding changes later, please let me know if you have a different opinion